analyzer/build.gradle

```diff
@@ -41,6 +41,7 @@ task funTest(type: Test) {
     group = 'Verification'
     classpath = sourceSets.funTest.runtimeClasspath
     testClassesDirs = sourceSets.funTest.output.classesDirs
+    systemProperties = System.properties
```

Comment:
We seem to be getting strange test failures with this. Maybe should not blindly forward all properties, but only known / required ones?

Reply:
On a somewhat related note, I've created https://github.com/kotlintest/kotlintest/issues/240